pino.js

```diff
@@ -220,6 +220,10 @@ Object.defineProperty(Pino.prototype, 'level', {
   set: Pino.prototype._setLevel
 })
 
+Object.defineProperty(Pino.prototype, 'lscache', {
+  value: copy({}, lscache)
```

Comment:
We don't use `lscache` anywhere else, so there's no need to copy it. This has exactly the same behavior like it was before.

Your tests only work because you delete the cache, which forces node to reinitialize the complete file.
`lscache` is still shared between multiple `pino` instances.